fairscale/optim/oss.py

```diff
@@ -285,17 +334,30 @@ def consolidate_state_dict(self, recipient_rank: int = 0) -> None:
                     if should_send_state
                     else torch.tensor([0], dtype=torch.uint8, device=dist_device)
                 )
-                broadcast_object(
-                    state_to_share, src_rank=self.global_rank, group=self.group, dist_device=dist_device,
-                )
+                if _gpu_capabilities_older_than_50():
```

Comment:
personally I would be fine dumping the old gpu case (with a proper warning), I'm not sure that the intersection of people willing to shard optimizer state and the people using old GPUs is that big ?

Reply:
yeah, I would be fine with that too. unfortunately, circle CI only have M60 for us for now. Maybe we will have T4 soon. Before that, we just have this local patch for now.